test/textpager_tests.c

```diff
@@ -8,15 +8,15 @@
 void new_tp_returns_non_null(void ** state){
```

Comment:
Should add tests to verify the scrolling.  Could just scroll the window and verify get_curses_cursor results are what they should be.